tests/dualtor/test_tor_ecn.py

```diff
@@ -105,12 +124,12 @@ def build_non_encapsulated_ip_packet(
     server_ipv4 = server_ips["server_ipv4"].split("/")[0]
     config_facts = tor.get_running_config_facts()
     try:
-        peer_ipv4_address = [dut_name["address_ipv4"] for dut_name in config_facts["PEER_SWITCH"].values()][0]
+        peer_ipv4_address = [_["address_ipv4"] for _ in config_facts["PEER_SWITCH"].values()][0]
     except IndexError:
         raise ValueError("Failed to get peer ToR address from CONFIG_DB")
 
-    tor_ipv4_address = [addr for addr in config_facts["LOOPBACK_INTERFACE"]["Loopback0"]
-                        if is_ipv4_address(addr.split("/")[0])][0]
+    tor_ipv4_address = [_ for _ in config_facts["LOOPBACK_INTERFACE"]["Loopback0"]
```

Comment:
If this code was previously working using `addr` I see no need to change it.

Reply:
Done